PV260_Project/BusinessLayer/Notifications/IEmailSender.cs

```diff
@@ -0,0 +1,15 @@
+﻿using BusinessLayer.Exceptions;
+using DataLayer.Models;
+
+namespace BusinessLayer.Notifications;
+
+public interface IEmailSender
+{
+    /// <summary>
+    /// Sends notification emails to the specified recipients regarding actual holding changes
+    /// </summary>
+    /// <param name="holdingChanges">list of holding changes</param>
+    /// <param name="recipients">recipients of given notification</param>
+    /// <exception cref="EmailSenderException">when sending notifications has failed</exception>
+    void SendDailyNotification(List<HoldingChanges> holdingChanges, List<Email> recipients);
```

Comment:
I wouldn't put the word `Daily` in the name of the method, I think it has nothing to do with the daily :D